src/main/java/SYSC6/Project/Main_Controller.java

```diff
@@ -26,60 +27,83 @@
 
     private Long id = 0L;
 
+    /**
+     * Sends the User to the login page once going to the Heroku Site
+     * @param model
+     * @return login_form (html page)
+     */
     @GetMapping("/")
     public String login(Model model){
         model.addAttribute("Login",new Login());
         return "login_form";
     }
 
+    /**
+     * If login successful it returns the id of the user and give this to the controller
+     * @param UserId
+     * @return user_portal page
+     */
     @PostMapping("/login_form")
-    public String login_process(@ModelAttribute Login login, Model model, RedirectAttributes attributes){
-        model.addAttribute("login", login);
-        ArrayList<User> users = checkUser();
-        for(User user : users){
-            if(user.getUsername().equals(login.getUsername())){
-                if(user.getPassword().equals(login.getPassword())){
-                    id = user.getId();
-                    return "redirect:/user_portal";
-                }
-            }
-        }
-        model.addAttribute("message", "Wrong login information");
-        return "login_form";
+    public String login_process(@RequestParam(value="id",required=true) String UserId){
+        id = Integer.parseInt(UserId) * 1L;
```

Comment:
just for my own clarification, this is the id associated with the user that is going to the user portal?